stdlib/public/core/Atomics/AtomicStorage.swift.gyb

```diff
@@ -0,0 +1,363 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Atomics open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+% from SwiftAtomics import *
+
+% for (type, size, alignment, builtin) in atomicTypes:
+
+//===----------------------------------------------------------------------===//
+// ${size} bit Atomic Storage
+//===----------------------------------------------------------------------===//
+
+@available(SwiftStdlib 5.10, *)
+@_alignment(${alignment})
+@frozen
+public struct ${type} {
+  @usableFromInline
+  var storage: ${builtin}
```

Comment:
Let's follow the Leading Underscore Rule here -- these types have non-underscored aliases.

```suggestion
  var _storage: ${builtin}
```

Additionally, it may be a good idea to expose `_storage` as public, emulating the existing public types 🤔

Reply:
Yeah, please make it public; this is important to allow people to prototype improvements in a package.